app/frontend/packs/autosuggests/init-autosuggest.js

```diff
@@ -26,3 +34,11 @@ export const initAutosuggest = ({ inputIds, containerId, templates = {}, styles
     console.error(`Could not enhance ${containerId}:`, err)
   }
 }
+
+initAutosuggest.stripWhitespaceFilter = (_source) => {
```

Comment:
Should `_source` have a leading underscore given that it is not an ignored param?

Reply:
Good point, I think I did that to avoid `source = source` below - this method only serves to create a closure for `source` so that it's accessible in the callback below. There may well be a better way to do this...